README.md

```diff
@@ -297,12 +297,15 @@ A number of options are available to allow you to customise the SDK:
       }
     },
     'document',
+    'poa',
```

Comment:
not sure how necessary it is to include it here, because we also don't include the `complete` step here. I guess it depends if this would be a normal flow or not

Reply:
Also, I'm not sure if we should call it `poa`, yet another acronym :/